tests/integration/ceph_csi_test.go

```diff
@@ -46,6 +46,9 @@ func runCephCSIE2ETest(helper *clients.TestClient, k8sh *utils.K8sHelper, s suit
 		t.Skip()
 	}
 
+	logger.Infof("temporarily skip the CSI tests")
```

Comment:
Is this intended to be in the PR?

Reply:
still investigating an issue related to the CSI tests, this is not intended to be merged...